app/src/main/java/com/github/sdpteam15/polyevents/model/database/remote/adapter/EventEditAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.sdpteam15.polyevents.model.database.remote.adapter
+
+import com.github.sdpteam15.polyevents.model.database.remote.DatabaseConstant
+import com.github.sdpteam15.polyevents.model.database.remote.DatabaseConstant.EventEditConstant.EVENT_EDIT_STATUS
+import com.github.sdpteam15.polyevents.model.entity.Event
+
+object EventEditAdapter : AdapterInterface<Event> {
+    override fun toDocument(element: Event): HashMap<String, Any?> {
+        val map = EventAdapter.toDocument(element)
+        map[EVENT_EDIT_STATUS.value] = element.status!!.ordinal
+        map[DatabaseConstant.EventEditConstant.EVENT_EDIT_ADMIN_MESSAGE.value] = element.adminMessage
+        map[DatabaseConstant.EventConstant.EVENT_DOCUMENT_ID.value] = element.eventId
+        map[DatabaseConstant.EventConstant.EVENT_DOCUMENT_ID.value] =element.eventId
+        return map
```

Comment:
duplicate EVENT_DOCUMENT_ID  = element.eventId
try to unify your static / non static imports

Reply:
Yep!